api.planx.uk/send/email.ts

```diff
@@ -35,7 +35,7 @@ const sendToEmail = async(req: Request, res: Response, next: NextFunction) => {
       const config: EmailSubmissionNotifyConfig = {
         personalisation: {
           emailReplyToId: notify_personalisation.emailReplyToId,
-          serviceName: "Test",
+          serviceName: payload?.flowName[0]?.toUpperCase() || "PlanX",
```

Comment:
When I tried this, `flowName` wasn't an array. This resulted in the string being truncated to just the first letter.

<img width="733" alt="image" src="https://user-images.githubusercontent.com/20502206/212847986-c0869240-5dcb-483c-83a6-dc3586c6e017.png">